### DISCUSSION TITLE
Skipping System Files

### DISCUSSION THREAD

**Discussion by amdanie8** [jonelo/jacksum#28] [Q&A] [answered]
Hello!

In using Jacksum to calculate checksums of backup HDDs, is there a method that people prefer to use in order to exclude system files from being checked while creating the checksum list?

Thanks!

**Comment by jonelo** [comment, marked as answer]
Thank you for your question! I am not quite sure what you actually mean by 'system files', but I have created a blog that describes [How to skip files during hash calculation or the verification process](https://loefflmann.blogspot.com/2023/09/jacksum-how-to-skip-files-during-hash-calculation-or-the-verification-process.html). Does that answer your question?

**Reply by amdanie8** [reply]
Hello sir! Thank you for your reply! Your blog post is certainly helpful and I appreciate the link. 

To clarify, I am performing some digital preservation work on a number of large (2+ TB) HDDs. In my initial tests using jacksum I asked the utility to create a hash list using one of the HDDs as the directory to scan. This resulted in a number of hang-ups and memory issues, because jacksum was allocating resources to files that were "not regular files" or (I believe) were specific to the HDD's architecture.  

With the commands you've provided in your blog post, I believe I can run hashes on these drives more efficiently. Thank you again!
